### ISSUE TITLE
Twitter style formatter doesn't work as advertised

### ISSUE THREAD

**Issue by danqing** [malcommac/SwiftDate#701]
Hi,

This is the same issue as #684 that I opened (since it has been closed). The Twitter style formatter doesn't work as advertised. As of v6.1.0 the issue is still present (and the test is testing the wrong string, as seen below). #684 has more pointers to the problematic changes. If this is intended, can you update the documentation? If it is not (and I hope it's not! because o/w Twitter style isn't really Twitter style anymore..) can you change it back? :)

https://github.com/malcommac/SwiftDate/blob/86644f41fa10b023b5e6f3f548a0dda9415f43b0/Tests/SwiftDateTests/TestFormatters.swift#L519

Thank you!

**Comment by danqing** [comment]
Hi @malcommac any thoughts on this? Thanks!

**Comment by lschwe** [comment]
This is pretty unfortunate. I'm literally using this pod only to format dates in twitter style...I guess gonna have to revert back to an older version.

**Comment by timothyerwin** [comment]
yup same here...shame this issue isn't handled properly...like no comments and not matching documentation

**Comment by vburojevic** [comment]
Having the same issue, not getting "3m" like in README.
For hours i.e. I get "11 hr. ago".

**Comment by vburojevic** [comment]
It seems that the JSON files don't even have that type of abbreviated format.
@malcommac was there a support for this before? Is it possible to add these translations back?

**Comment by balazserdeszlogmein** [comment]
This last worked fine in 6.0.2. You need reference that version, which is already pretty unfortunate.

But even if you get that version, twitterStyle is bugged as per this issue: #579

**Comment by martindufort** [comment]
Looks like the switch from 6.0.2 to 6.0.3 to `.json` files for languages actually removed the `.tiny` specification which is used by the `.twitter` flavor. 

There is no `tiny` specifications in any of the JSON language files...
